packages/preview/bubble/0.2.0/lib.typ

```diff
@@ -0,0 +1,126 @@
+// main project
+#let bubble(
+  title: "",
+  subtitle: "",
+  author: "",
+  affiliation: "",
+  year: none,
+  class: none,
+  date: datetime.today().display(),
+  logo: none,
+  main-color: "E94845",
+  alpha: 60%,
+  color-words: (),
+  body,
+) = {
+  set document(author: author, title: title)
+
+  // Save heading and body font families in variables.
+  let body-font = "Source Sans Pro"
+  let title-font = "Barlow"
+
+  // Set colors
+  let primary-color = rgb(main-color) // alpha = 100%
+  // change alpha of primary color
+  let secondary-color = color.mix(color.rgb(100%, 100%, 100%, alpha), primary-color, space:rgb)
+
+  // highlight important words
+  show regex(color-words.join("|")): text.with(fill: primary-color)
```

Comment:
If `color-words` is an empty array, this will crash (`().join("|")` returns `none` and not a string), which will probably happen often as this is the default value of the argument. If this is expected, please tell me, otherwise you can probably only apply this show rule if the array is not empty.